docs/HTTP-streamingsource.md

```diff
@@ -0,0 +1,425 @@
+| type            | /fields/issuetype/name                    |
+| description     | /fields/description                       |
+| projectCategory | /fields/project/projectCategory/name      |
+| isSubtask       | /fields/issuetype/subtask                 |
+| fixVersions     | /fields/fixVersions                       |
+
+The result records are:
+
+| key	     | type	       | isSubtask | description	                 | projectCategory	| fixVersions |
+| -------- | ----------- | --------- | ----------------------------- | ---------------- | ----------- |
+| NETTY-14 | Bug         | false     | Test description for NETTY-14 | Infrastructure	  | ["4.1.37"]  |
+| NETTY-13 | Improvement | false     | Test description for NETTY-13 | Infrastructure   |	[]          |
+
+Note, that field `key` was mapped without being included into the mapping. Mapping entries like `key: /key`
+can be omitted as long as the field is present in schema.
+<br>
+
+**CSV Skip First Row:** Whether to skip the first row of the HTTP response. This is usually set if the first row is a header row.
+
+### Basic Authentication
+
+**Username:** Username for basic authentication.
+
+**Password:** Password for basic authentication.
+
+### HTTP Proxy
+
+**Proxy URL:** Proxy URL. Must contain a protocol, address and port.
+
+**Username:** Proxy username.
+
+**Password:** Proxy password.
+
+### Error Handling
+
+**HTTP Errors Handling:** Defines the error handling strategy to use for certain HTTP response codes.
+The left column contains a regular expression for HTTP status code. The right column contains an action which
+is done in case of match. If HTTP status code matches multiple regular expressions, the first specified in mapping
+is matched.
+
+Example:
+
+| HTTP Code Regexp  | Error Handling          |
+| ----------------- |:-----------------------:|
+| 2..               | Success                 |
+| 401               | Retry and fail          |
+| 4..               | Fail                    |
+| 5..               | Retry and skip          |
+| .*                | Fail                    |
+
+Note: pagination types "Link in response header", "Link in response body", "Token in response body" do not support
+"Skip", "Retry and skip" options.
+
+**Non-HTTP Error Handling:** Error handling strategy to use when the HTTP response cannot be transformed to an output record.
+Possible values are:<br>
+Stop on error - Fails pipeline due to erroneous record.<br>
+Skip on error - Ignores erroneous records.
+
+**Retry Policy:** Policy used to calculate delay between retries.
+
+**Linear Retry Interval:** Interval between retries. Is only used if retry policy is "linear".
+
+**Max Retry Duration:** Maximum time in seconds retries can take.
+
+**Connect Timeout:** Maximum time in seconds connection initialization is allowed to take.
+
+**Read Timeout:** Maximum time in seconds fetching data from the server is allowed to take.
+
+### Pagination
+
```

Comment:
There should be some general information here about how the plugin behaves when there is pagination vs when there isn't. 

When there is pagination, it reads pages in order, keeping track of which pages have already been read. It will not reprocess any page it has already read.

When there is no pagination, it periodically polls the same page.

Reply:
Done